### PR TITLE
Release 1.20.1 - fix DeltaResponseHandler

### PR DESCRIPTION
**Pull request by MIchaelMainer**
Contains the fix for DeltaResponseHandler and CI/CD updates.

This will result in a release for 1.21.1.

@andrueastman I suggest you add a task to cherry pick c392714.

### CODE REVIEW

File: src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj [hunk 1]
```diff
@@ -19,12 +19,10 @@
     <DelaySign>false</DelaySign>
     <AssemblyOriginatorKeyFile>35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <VersionPrefix>1.20.0</VersionPrefix>
+    <VersionPrefix>1.20.1</VersionPrefix>
```
Comment (ddyett) [inline src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj]: i think 1.20.1 is correct but the title is indicating this should be 1.21.1

Comment (MIchaelMainer) [reply src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj]: Fixed that. https://www.nuget.org/packages/Microsoft.Graph.Core 1.20.0 is the last release for this major version.

**Comment by MIchaelMainer** [comment]
- [ ] Re-enable tests for all of these yaml pipelines.

**Comment by MIchaelMainer** [comment]
One last change, I swear. Need to enable tests on building for preview and production build pipelines.

**Pull request merged**